nautobot/utilities/testing/views.py

```diff
@@ -448,6 +450,29 @@ def test_create_object_with_constrained_permission(self):
             else:
                 self.assertInstanceEqual(self._get_queryset().last(), self.form_data)
 
+        def test_slug_autocreation(self):
+            """Test that slug is autocreated through ORM."""
+            # This really should go on a models test page, but we don't have test structures for models.
+            if self.slug_source is not None:
+                filter = self.slug_source + "__contains"
+                object = self.model.objects.filter(**{filter: self.slug_test_object})[0]
```

Comment:
What's the reason we need to use `filter()` and `__contains` instead of just directly getting the specified object (i.e. `object = self.model.objects.get(**{self.slug_source: self.slug_test_object})`)?

Reply:
`filter()` was what I could get to work, I will test switching to trying to get it instead. I wasn't sure if the get on self.slug_source would work when passing a string instead of object on the self.slug_source.